api/src/test/java/org/openmrs/PersonTest.java

```diff
@@ -375,7 +375,6 @@ public void getAge_shouldGetAgeAfterDeath() {
 		deathDate.set(2000, Calendar.JUNE, 3);
 		Person person = new Person();
 		person.setBirthdate(birthdate.getTime());
-		person.setDead(true);
```

Comment:
The behavior of Person#setDead hasn't been changed. 
I deleted setDead calls from unrelated tests like Person#getAge(Date) and PatientService#mergePatients(Patient, Patient). 
How do you think, maybe we need a special test for Person#setDead call?
If so, should I make it in the same PR, or make a separate task in Jira?